src/transformer/schema-transformer.ts

```diff
@@ -127,35 +149,51 @@ export class SchemaTransformer {
         ],
       },
     };
+
+    this.transformConfig = props.transformConfig ?? (this.isSyncEnabled ? this.loadConfigSync() : {});
+    this.amplifyTransformers = props.amplifyTransformers ?? [new DynamoDBModelTransformer(),
+      new TtlTransformer(),
+      new VersionedModelTransformer(),
+      new FunctionTransformer(),
+      new KeyTransformer(),
+      new ModelConnectionTransformer(),
+      new ModelAuthTransformer(this.authTransformerConfig),
+      new HttpTransformer(),
+      new CustomVTLTransformer(this.customVtlTransformerRootDirectory)];
```

Comment:
This should go back to where it was, outside of the "amplifyTransformers" as it's a custom one not supported by amplify